opentelekomcloud/resource_opentelekomcloud_lb_listener_v2.go

```diff
@@ -75,12 +75,13 @@ func resourceListenerV2() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
-				ForceNew: true,
+				ForceNew: true, // could be updated duo to docs, but gopher doen´t define it
 			},
 
 			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
+				ForceNew: false, // could be updated
```

Comment:
If this can be updated, then why make it `ForceNew`?

Reply:
> If possible, please also move `opentelekomcloud/util.go` to another PR.
> Ideally, this should be 3 PRs, but it's more or less ok for lb resources to be a part of a single one.

Thought about this before. The problem is that the acceptance tests of this PR will not work without the util.go patch.
I don´t know how to define dependent PR. If you can give me a hint...

Otherwise, it is maybe not a completely clean separation of concerns, but there is no obvious obstacle with keeping both together.

Reply:
You can make `opentelekomcloud/util.go` PR first. If I remember correctly, you can make PR dependent adding `Depends-On: <PR link>` to PR description - zuul should process that